src/requestHandler.ts

```diff
@@ -1022,9 +1022,7 @@ export default class RequestHandler {
         limit: MaximumRequestSize,
       })
     );
-    this.api.use(
-      bodyParser.raw({ type: "application/*", limit: MaximumRequestSize })
-    );
+    this.api.use(bodyParser.raw({ type: "*/*", limit: MaximumRequestSize }));
```

Comment:
I'm not sure why I'd originally restricted using the `raw` body parser to only `application/*` content-types, but here I've loosened this up such that it'll always use the `raw` parser if the incoming content type wasn't processed by an earlier parser.